packages/server/graphql/mutations/helpers/endMeeting/sendNewMeetingSummary.ts

```diff
@@ -10,12 +11,17 @@ export default async function sendNewMeetingSummary(newMeeting: Meeting, context
   if (summarySentAt) return
   const now = new Date()
   const r = await getRethink()
-  const {dataLoader} = context
-  const [teamMembers, team] = await Promise.all([
+  const {dataLoader, authToken} = context
+  const viewerId = getUserId(authToken)
+  const [teamMembers, team, user] = await Promise.all([
     dataLoader.get('teamMembersByTeamId').load(teamId),
     dataLoader.get('teams').loadNonNull(teamId),
+    dataLoader.get('users').load(viewerId),
     r.table('NewMeeting').get(meetingId).update({summarySentAt: now}).run()
   ])
+  if (!user) return
+  const {sendSummaryEmail} = user
+  if (sendSummaryEmail === false) return
```

Comment:
-2 I did not test, but as I understand the setting it allows each user to set their preferences themselves. In the current implementation only the setting of the viewer who ends the meeting is considered, not of any other meeting participant. The check should be down in line 33.
```ts
  const emailAddresses = users.filter(isValid).filter(({sendSummaryEmail}) => sendSummaryEmail !== false).map(({email}) => email)
```

Reply:
Good catch, thank you!